lib/rpmrc.c

```diff
@@ -1039,6 +1039,8 @@ static void defaultMachine(rpmrcCtx ctx, const char ** arch, const char ** os)
 	    strcpy(un.machine, "i386");
 #elif defined(__x86_64__)
 	    strcpy(un.machine, "x86_64");
+#elif defined(__arm64__)
```

Comment:
The preferred define from ARM is `__aarch64__`, can you change to that?

Reply:
Sure, done.